Extension/ui/settings.html

```diff
@@ -552,7 +556,7 @@
                 <option value="c11">c11</option>
                 <option value="c99">c99</option>
                 <option value="c89">c89</option>
-                <option value="gnu18">gnu18</option>
+                <option value="gnu17">gnu18</option>

```

Comment:
Why was this changed? Can you change this back? We decided to just expose c18/gnu18 in settings, although we'll accept -std=c17/gnu17 via the compilerArgs (which will convert it to c18 internally).

Reply:
Okk